libs/util/shared/src/lib/models/search.model.ts

```diff
@@ -54,11 +54,13 @@ export interface RecordMetric {
   recordCount: number
 }
 
-export enum ResultsListLayout {
+export enum ResultsListLayoutEnum {
   CARD = 'CARD',
   LIST = 'LIST',
   TEXT = 'TEXT',
   TITLE = 'TITLE',
 }
 
+export type ResultsListLayout = ResultsListLayoutEnum | string
```

Comment:
This feels weird. Why keep the enum there since the layout configuration is now in `ui/search` and can be overridden in apps?

I think in the state the result layout should just be a string, and the enum should be removed altogether. Then we can handle an incorrect layout value to avoid any unintentional misconfiguration.